user-journeys/replay_on_gcp.sh

```diff
@@ -44,7 +44,9 @@ echo "Create a Cloud Run job"
 gcloud beta run jobs create user-journeys-demo \
   --tasks $no_of_journeys \
   --image us-central1-docker.pkg.dev/${PROJECT_ID}/containers/user-journeys-demo:latest \
-  --service-account no-permission@${PROJECT_ID}.iam.gserviceaccount.com
+  --service-account no-permission@${PROJECT_ID}.iam.gserviceaccount.com \
+  --task-timeout 3600 \
```

Comment:
I'm nor sure we want to default to 1h timeout. It's probably better to still enforce a reasonable timeout. The default is 10minute, I'm not sure we should expect user journeys that takemore than 10min to complete.

I suggest removing this line